models.py

```
@@ -0,0 +1,36 @@
+import json
+
+
+class JSONModel:
+    def __init__(self, json_file="scratch.json"):
+        self.json_file = json_file
+        try:
+            with open(self.json_file, 'r') as data_file:
+                self.data = json.load(data_file)
+        except (FileNotFoundError, json.decoder.JSONDecodeError):
```

We may want to throw an exception if we're unable to load the data, otherwise we have no way of knowing if that failed but not sure if it matters for your example.